network-api/networkapi/wagtailpages/migrations/0008_auto_20200819_2044.py

```diff
@@ -10,7 +10,7 @@ class Migration(migrations.Migration):
     dependencies = [
         ('wagtailimages', '0022_uploadedimage'),
         ('wagtaildocs', '0010_document_file_hash'),
-        ('wagtailpages', '0006_publicationpage'),
+        ('wagtailpages', '0007_articlepage'),
```

Comment:
this has not landed yet, as far as I know?

Reply:
correct, this would merge into the feature branch we have for publication pages: https://github.com/mozilla/foundation.mozilla.org/pull/5098

Reply:
never mind, not targetting `master` =D